src/main/scala/rules/ChunkSupporter.scala

```diff
@@ -26,12 +26,12 @@ trait ChunkSupportRules extends SymbolicExecutionRules {
               ve: VerificationError,
               v: Verifier,
               description: String)
-             (Q: (State, Heap, Term, Verifier) => VerificationResult)
-             : VerificationResult
+             (Q: (State, Heap, Term, Verifier) => VerificationResultWrapper)
```

Comment:
Changing from `VerificationResult` to `VerificationResultWrapper` affects loads of public APIs and may force clients/frontends to perform similar boilerplate changes.

Please think about changing `VerificationResult` itself — would this be a reasonable alternative?